static/css/character_card_manager.css

```
@@ -7121,6 +7242,800 @@ margin-top: 10px;
+    box-shadow: 0 18px 44px rgba(64, 197, 241, 0.24);
+    display: flex;
+    flex-direction: column;
+    z-index: 10100;
+    font-family: inherit;
+    overflow: hidden;
+    animation: cardCompanionWindowIn 0.22s ease-out;
+    transition: box-shadow 0.22s ease, background 0.22s ease, opacity 0.22s ease;
+    resize: both;
+}
+
+@keyframes cardCompanionWindowIn {
```

Stylelint (17.12.0)

[error] Expected keyframe name "cardCompanionWindowIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)